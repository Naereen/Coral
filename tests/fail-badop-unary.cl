x = 3

if x == 3:
	b = "a"

print(7)

c = -"a"

print(10)

