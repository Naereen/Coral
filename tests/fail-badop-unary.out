7
invalid use of - operator
